atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/AbstractCassandraKeyValueServiceIntegrationTest.java

```diff
@@ -144,7 +129,7 @@ protected boolean reverseRangesSupported() {
     }
 
     @Override
-    @Ignore
+    @Disabled
```

Comment:
This causes test count to be decrease by 2 (since we have two parameters). `@Ignore` was not working as expected in the previous implementation (the test was working anyway) so it should be okay now (the test does not work).